fastlane/Fastfile

```diff
@@ -47,7 +47,6 @@ platform :ios do
       scheme: "UnitTests",
       testplan: "UnitTests",
       prelaunch_simulator: true,
-      devices: ["iPhone 8", "iPhone 12"],
```

Comment:
this setting is also present in line 46, but that one actually specifies the OS version. 
It looks like this was a rebase artifact, since the original PRs for both lines seem to be fine. ¯\\_(ツ)_/¯